crates/core/tedge_mapper/src/c8y/service_monitor.rs

```diff
@@ -50,36 +46,42 @@ impl TopicInfo {
     }
 }
 
-pub fn convert_health_status_message(message: &Message, device_name: String) -> Vec<Message> {
+pub fn convert_health_status_message(
+    message: &Message,
+    device_name: String,
+    service_type: String,
```

Comment:
```suggestion
    default_service_type: String,
```
Just to be clearer as I was really struggling to follow the code below with the combination of `service_type`, `health_status.service_type` and `s_type` in the same context ;-)

Reply:
fixed